.travis.yml

```diff
@@ -1,4 +1,10 @@
 language: rust
+os:
+    - linux
+    - windows
+    - osx
 rust:
+    - 1.31.0
```

Comment:
anything particular about 1.31.0?

Reply:
Nope, but I typically include the oldest version of Rust I'm willing to support in the CI. I guess this should be whatever min version you think makes sense?

Reply:
ah makes sense, 1.31.0 looks good as the min version of 2018 edition